packages/cli/src/commands/env/pull.ts

```diff
@@ -125,6 +136,12 @@ export default async function pull(
     )}\n`
   );
 
+  if (deltaString) {
+    output.print('\n' + deltaString);
+  } else if (exists) {
+    output.log('No changes found.');
```

Comment:
If `createEnvObject` throws an error, it will end up returning `undefined`. When it gets down here, there could have been actual changes, but we'll log here that there were none. In this case, I think it's better to either say nothing or be clear that we couldn't detect the changes, but the latest values were updated.